freqtrade/resolvers/strategy_resolver.py

```diff
@@ -111,6 +111,12 @@ def _strategy_sanity_validations(self):
             raise ImportError(f"Impossible to load Strategy '{self.strategy.__class__.__name__}'. "
                               f"Order-time-in-force mapping is incomplete.")
 
+        # Stoploss on exchange does not make sense, therefore we need to disable that.
+        if config.get('dry_run'):
```

Comment:
Check here for RunMode instead? (all modes where it's needed: dry-run, backtesting, hyperopt)...?

In this case same code in backtesting can be removed, I guess...

Reply:
good point :+1: